.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
In the master branch, create an initial/base commit and commit-push git-related files, base setup
Ideally, this PR should only have the changes related to story/task

Reply:
ok